testing/testcafe/tests/navigation/form/form.labelMode.ts

```diff
@@ -0,0 +1,222 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`Form`
+  .page(url(__dirname, '../../container.html'));
+
+[undefined, true, false].forEach((labelVisible) => {
+  ['outside', 'floating', 'hidden', 'static'].forEach((formLabelMode) => {
+    [undefined, 'floating', 'hidden', 'static'].forEach((editorLabelMode) => {
```

Comment:
A lot of screenshot tests is risky: something might/will fail on style/size/browser/defaults changes.
Have you considered something like "assert.equal($('#form').find('.dx-field-item .dx-textbox').dxTextBox('instance').option("labelMode"), "hidden")" in UT? It checks formLabel dom element and dxTextBox properties and hasn't that risk, it's easy to review it in action (when debugging), it checks what our code do.

Reply:
Yep, great idea - cover with qunit tests